recipes/r-bedtoolsr/meta.yaml

```diff
@@ -5,7 +5,7 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/PhanstielLab/bedtoolsr/archive/refs/tags/v2.30.0-2.tar.gz
+  url: https://github.com/PhanstielLab/bedtoolsr/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
The upstream version is `2.30.0-2`, but conda recipe versions cannot contain `-` apparently, which is why I changed the recipe version to `v2.30.0.2`.

Reply:
good catch